assets/csidriveroperators/vsphere/06_clusterrole.yaml

```diff
@@ -309,6 +309,8 @@ rules:
   - get
   - list
   - watch
+  - patch
+  - update
```

Comment:
This allows the vSphere CSI driver operator to patch any pod and ConfigMap in any namespace. IMO it's too broad.
It should allow only selected ConfigMap in openshift-config-managed namespace, i.e. a Role could be better. And definitely decoupled from Pods.

Reply:
Moved this to use named configmap. I think that works as well.